ts/output/chtml/FontData.ts

```diff
@@ -370,9 +372,9 @@ export class CHTMLFontData extends FontData<CHTMLCharOptions, CHTMLVariantData,
 
   /**
    * @param {StyleList} styles  The style object to add styles to
-   * @param {string} vclass     The variant class string (e.g., .mjx-b) where this character is being defined
+   * @param {string} vletter    The variant class string (e.g., .mjx-b) where this character is being defined
```

Comment:
The description is no longer accurate. This is now the variant class letter (e.g., 'B' for bold, 'SS for sans-serif).